recipes/guetzli/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, MSBuild, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class GoogleGuetzliConan(ConanFile):
+    name = "guetzli"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://opensource.google/projects/guetzli"
+    description = "Perceptual JPEG encoder"
+    topics = "jpeg", "compression"
+    exports_sources = "patches/**"
+    settings = "os", "compiler", "build_type", "arch"
```

Comment:
```suggestion
    settings = "os", "compiler", "arch"
```

It's an installer package, doesn't require build type. It will be used as build_requirement for 99% of cases.